doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1150,6 +1150,15 @@ Please note that there are other reasons why download URLs may not be included f
 
 For more on Schema.org JSON-LD, see the :doc:`/admin/metadataexport` section of the Admin Guide.
 
+http.request-timeout-seconds
+++++++++++++++++++++++++++++
+
+To facilitate large file upload and download, the Dataverse installer bumps the Payara **server-config.network-config.protocols.protocol.http-listener-1.http.request-timeout-seconds** setting from its default 900 seconds (15 minutes) to 3600 (1 hour). Should you wish to shorten or lengthen this window, issue:
+
+``./asadmin set server-config.network-config.protocols.protocol.http-listener-1.http.request-timeout-seconds=nnn``
```

Comment:
```suggestion
``./asadmin set server-config.network-config.protocols.protocol.http-listener-1.http.request-timeout-seconds=1800``
```

I would rather see this as real value (1800) than something you have to interpret (nnn).

Reply:
also done.